afm/asset.py

```diff
@@ -2,13 +2,18 @@
 # Copyright 2020 IBM Corp.
 # SPDX-License-Identifier: Apache-2.0
 #
+import httpfs
 import os
 
 from afm.config import Config
 from afm.pep import registry, consolidate_actions
 from afm.s3 import s3filesystem_from_config
 
-from pyarrow.fs import LocalFileSystem
+from pyarrow.fs import LocalFileSystem, PyFileSystem, FSSpecHandler
```

Comment:
Please move out of asset.py. We can create a filesystem package if needed.

Reply:
done